roles/setup/tasks/configuration.yml

```diff
@@ -6,36 +6,36 @@
     repo: https://github.com/robbyrussell/oh-my-zsh.git
     dest: "{{general.local_home}}/.oh-my-zsh"
```

Comment:
zsh as default shell, with oh-my-zsh, is probably controversial from a user standpoint, but yolo.